src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Formatting/HtmlFormattingPass.cs

```diff
@@ -57,7 +57,8 @@ public async override Task<FormattingResult> ExecuteAsync(FormattingContext cont
 
             var originalText = context.SourceText;
 
-            var htmlEdits = await HtmlFormatter.FormatAsync(context, context.Range, cancellationToken);
+            var range = TextSpan.FromBounds(0, originalText.Length).AsRange(originalText);
+            var htmlEdits = await HtmlFormatter.FormatAsync(context, range, cancellationToken);
```

Comment:
I forget does this end up calling through to the client to resolve HTML edits or do we get HTML formatter edits in-proc?

Asking because if it's the latter we could probably change the end-to-end (at least on Razor side) HTML formatting stack to not take a range

Reply:
Yeah, this calls the html formatter via LSP

Reply:
How do you feel about removing the Range bit entirely from `HtmlFormatter.FormatAsync`? Not sure if there's other demons with that approach but I figure if there's issues calling range formatting on HTML in Razor then might as well not allow it 😄

Reply:
Yeah, I can move the range calculation inside that method, that makes sense.

Reply:
Instead of doing a range formatting request to HTML should we just do a doc formatting request?

Reply:
You want me to make a whole new endpoint!? 😛